### PR TITLE
Feat/388 remove bootstrap

### LINKED ISSUE #388: Switch bootstrap with tailwindcss
Tailwind makes it easier to define custom classes and purge unused classes than bootstrap which means we will have smaller builds.

- [x] configure webpack to use tailwindcss
- [x] make sure hot reloading works when css file is changed
- [x] redesign components using tailwindcss classes instead of bootstrap
- [ ] deprecate bootstrap

Configuration already available at feat/redesign branch

### PR DESCRIPTION
**Pull request by ShynRou**
Resolves #388 finally.

- Removed react-bootstrap & react-bootstrap-icons
- Removed bootstrap
- Replaced all bootstrap classes I could find with tailwind

Some font sizes have slightly changed otherwise all should be as it was.
@JoeStorm99 Please test against test.sovryn.app and note any noticeable changes.

### CODE REVIEW

File: src/app/components/ActiveUserLoanContainer/components/ExpandedRowMobile/index.tsx [hunk 1]
```diff
@@ -15,12 +15,12 @@ export function ExpandedRowMobile(props) {
 
   return (
     <div style={{ opacity: '1' }} onClick={props.handleClick}>
-      <div className="tw-grid tw-gap-8 tw--mx-4 tw-grid-cols-12 table-header sub-header">
```
Comment (creed-victor) [inline src/app/components/ActiveUserLoanContainer/components/ExpandedRowMobile/index.tsx:18]: I think `tw--mx-4` means `-mx-4` (negative 1rem), was it decided to change to positive margin?

**Review by creed-victor — COMMENTED** [review]
Except for `tx--mx-*` changing `tx-mx-*` everything looks fine. Can approve PR if it was intended change (just need to go through entire website to check how everything changed visually too)

**Comment by ShynRou** [comment]
@creed-victor A damn it. Nope that's an error on my side… I'll try and fix that :/

File: src/app/components/ActiveUserLoanContainer/components/ExpandedRowMobile/index.tsx (hunk 1 above)
Comment (ShynRou) [reply src/app/components/ActiveUserLoanContainer/components/ExpandedRowMobile/index.tsx]: Fixed it.

**Comment by creed-victor** [comment]
- [ ] Dropdown in swap page is not full width, arrow is out of place:

Screenshot 2021-07-22 at 11 04 15

- [ ] Dropdown in borrow page has no border/background, not sure if worth fixing as entire page should be reworked, but maybe component is reused somewhere else.

Screenshot 2021-07-22 at 11 08 12

- [ ] Rewards page - button label is in lowercase

Screenshot 2021-07-22 at 11 12 01

**Comment by ShynRou** [comment]
@creed-victor Fixed the first and second issue, the third is the same as the state on production.

**Review by creed-victor — DISMISSED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Pull request merged**